dvl-a50/mavlink2resthelper.py

```diff
@@ -166,24 +169,29 @@ def __init__(self):
 }}
 """
 
-    def get_float(self, path: str) -> float:
+    def get_float(self, path: str, vehicle: int=None, component: int=None) -> float:
         """
-        Helper to get mavlink data from mavlink2rest
+        Helper to get mavlink data from mavlink2rest.
+        Uses initialised vehicle and component as defaults, unless overridden.
         Example: get_float('/VFR_HUD')
-        Returns the data as a float or False on failure
+        Returns the data as a float (nan on failure)
         """
-        response = request(MAVLINK2REST_URL + '/mavlink' + path)
+        response = self.get(path, vehicle, component)
         if not response:
             return float("nan")
         return float(response)
 
-    def get(self, path: str) -> str:
+    def get(self, path: str, vehicle: int=None, component: int=None) -> str:
```

Comment:
`vehicle` and `component` here are `Optional`, and we should maybe also make the return type `Optional[str]` as well, and change the failure response from `False` to `None` (or alternatively keep the return type and change the failure response to an empty string).
```suggestion
    def get(self, path: str, vehicle: Optional[int]=None, component: Optional[int]=None) -> Optional[str]:
```